Desktop/mainwindow.cpp

```diff
@@ -510,8 +513,10 @@ void MainWindow::loadQML()
 		ActiveModules::getActiveCommonModules(),
 		ActiveModules::getActiveExtraModules());
 
```

Comment:
I guess something went wrong with the rebasing, but there should be closing `}` here.

Reply:
> I think the Engine Overview form needs some work,
>       * A normal font size should do the job too

A bit strange the font is so huge for you though, are you zooming?

>     * The form probably doesn't need the maximize button, and it can be a Dialog

I guess so, but it also doesnt matter really.

>     * The form can be called several times. I think this should not be allowed.

I agree, but I dont find it important. If I have some time I can see if I can at something to avoid that.
 
> 
> I noticed that after some times, Engines will be killed, and upon interaction they come back to life. I guess this is something we do, but I've never noticed it!

Well, this PR changes a lot of things in the behaviour of engines ;)
This is one of the main goals actually :p

>     * Those circles are massive, and can probably be a bit smaller, and minimal
> the screenshot you show
Hmm the circles are much larger relative to the text for you then for me. 

![image](https://user-images.githubusercontent.com/29062713/166522990-264e31c8-0ca5-4c81-9198-6af033050ec4.png)
Is what it looks like in default width of the form